comictaggerlib/resulttypes.py

```diff
@@ -85,6 +86,7 @@ class CVVolume(TypedDict):
 
 
 class CVIssuesResults(TypedDict):
+    source: str
```

Comment:
These CV result classes exist to model the json response from comicvine so should not have the source attribute added.